src/reducers/filter.js

```diff
@@ -9,13 +9,10 @@ function handleCheckFilter(state, groupName, tagName){
   if(!state.hasOwnProperty(groupName) ||
     !state[groupName].hasOwnProperty(tagName))return state;
 
-  // Create next state and get state of checkbox
-  const nextState = {...state};
+  // Create next state
   const checked = state[groupName][tagName];
+  return {...state, ...{[groupName]: {...state[groupName], [tagName]: !checked}}};
```

Comment:
Du kan kopiere state først, så endre den property du ønsker. Litt enklere å lese, for meg. Feks:

newState = {...state}
newState[groupName][tagName] = !checked